### PR TITLE
@facebook-github-bot feature is now a bit nicer to people

### PR DESCRIPTION
**Pull request by bestander**
@mkonicek how can I add a line break to that?

### CODE REVIEW

File: bots/IssueCommands.txt [hunk 1]
```diff
@@ -36,5 +36,5 @@ comment Okay, reopening this issue.
...
 
 @facebook-github-bot feature
-comment Hey {issue_author}! Thanks for opening the issue, however it looks like a feature request. As noted in the [Issue template](https://github.com/facebook/react-native/blob/master/ISSUE_TEMPLATE.md) we'd like to use the GitHub issues to track bugs only. Can you implement the feature as a standalone npm module? If not consider sending a pull request or a create an entry on [Product Pains](https://productpains.com/product/react-native). It has a voting system and if the feature gets upvoted enough it might get implemented. Will close this issue.
+comment Hey {issue_author}! Thanks for opening the issue, however it looks like a feature request. As noted in the [Issue template](https://github.com/facebook/react-native/blob/master/ISSUE_TEMPLATE.md) we'd like to use the GitHub issues to track bugs only. Can you implement the feature as a standalone npm module? If not consider sending a pull request or a create an entry on [Product Pains](https://productpains.com/product/react-native). It has a voting system and if the feature gets upvoted enough it might get implemented. Closing this now due to the explanation above, thanks for understanding!
```
Comment (satya164) [inline bots/IssueCommands.txt]: `Closing this now due to the explanation above` sounds wrong?

**Comment by mkonicek** [comment]
Looks good, thanks!

Don't know how to add linebreaks actually.

> Closing this now due to the explanation above sounds wrong?

Yeah, maybe just:

> Closing this now, thanks for understanding!

**Comment by bestander** [comment]
@facebook-github-bot shipit

**Comment by bestander** [comment]
let me know if there is anything to improve

**Pull request closed without merging**